engine/engine_manifest.json

```
@@ -4,4 +4,11 @@
     "brand_name": "VOICEVOX",
     "url": "https://github.com/VOICEVOX/voicevox_engine",
     "default_sampling_rate": 24000
```

,抜けてた。ありがとうsuper linter。。
```suggestion
    "default_sampling_rate": 24000,
```